Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/commander.adoc

= Commander application
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Core Commander application setup describes all options including pre-install, install & post-install to manage your Lisk Core node.

Reviewer: ```suggestion
:description: Describes all requirements and dependencies to setup a Lisk Core node with Lisk Commander.
```
